src/test/java/io/cdap/plugin/MockEmitter.java

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
Can't we use this instead?
https://github.com/cdapio/cdap/blob/develop/cdap-app-templates/cdap-etl/hydrator-test/src/main/java/io/cdap/cdap/etl/mock/common/MockEmitter.java

Reply:
yes we should fixed.